pymobiledevice3/cli/remote.py

```diff
@@ -165,11 +154,39 @@ def cli_start_tunnel(udid: str, secrets: TextIO, script_mode: bool, max_idle_tim
 
     if udid is not None and rsd.udid != udid:
         raise NoDeviceConnectedError()
+    return rsd
 
+
+@remote_cli.command('start-tunnel')
+@click.option('--udid', help='UDID for a specific device to look for')
+@click.option('--secrets', type=click.File('wt'), help='TLS keyfile for decrypting with Wireshark')
+@click.option('--script-mode', is_flag=True,
+              help='Show only HOST and port number to allow easy parsing from external shell scripts')
+@click.option('--max-idle-timeout', type=click.FLOAT, default=MAX_IDLE_TIMEOUT,
+              help='Maximum QUIC idle time (ping interval)')
+@click.option('-p', '--protocol', type=click.Choice([e.value for e in TunnelProtocol]),
+              default=TunnelProtocol.QUIC.value)
+@sudo_required
+def cli_start_tunnel(udid: str, secrets: TextIO, script_mode: bool, max_idle_timeout: float, protocol: str):
+    """ start quic tunnel """
+    protocol = TunnelProtocol(protocol)
+    if not verify_tunnel_imports():
+        return
+    rsd = select_device(udid)
     asyncio.run(tunnel_task(rsd, secrets, script_mode, max_idle_timeout=max_idle_timeout, protocol=protocol),
                 debug=True)
 
 
+@remote_cli.command('delete-pair')
+@click.option('--udid', help='UDID for a specific device to delete the pairing record of')
+@sudo_required
```

Comment:
it doesn't require

Reply:
It does, in order to delete the pair record created by sudo